ui/zenoedit/nodesys/zenosubgraphscene.cpp

```diff
@@ -512,8 +521,55 @@ void ZenoSubGraphScene::mousePressEvent(QGraphicsSceneMouseEvent* event)
 
 void ZenoSubGraphScene::mouseMoveEvent(QGraphicsSceneMouseEvent* event)
 {
-    if (m_tempLink) {
+    if (m_tempLink)
+    {
+        bool bFixedInput = false;
+        QString nodeId;
+        m_tempLink->getFixedInfo(nodeId, QString(), QPointF(), bFixedInput);
```

Comment:
cannot bind non-const lvalue reference of type ‘QString&’ to an rvalue of type ‘QString’